latte/cs/macros.texy

```diff
@@ -776,4 +796,18 @@ Dumpování proměnných `{dump}` .{toc: Dumpování proměnných}
 {dump}       {* Vypíše všechny aktuálně definované proměnné *}
 \--
 
+
+
+Operátor rozbalení pole `(expand)` .{toc: Operátor rozbalení}
+----------------
+
+Operátor `(expand)` zabraňuje obalení argumentu prázdným polem v makrach, které očekávají rozbalené pole, jako  `{include}` a `n:attr`.
+Je užitečný pro posílání argumentů do vkládaných bloků nebo šablon a pro generování více atributů z pole.
```

Comment:
Operátor `(expand)` umožňuje použití pole v místě, kde se očekává více argumentů. Jde o obdobu operátoru `...` z PHP (odkaz na http://php.net/manual/en/functions.arguments.php#functions.variable-arg-list.new), na rozdíl od něj zachovává i klíče.

Lze jej použít například pro předání argumentů do bloků nebo vkládaných šablon, pokud je máme v poli.